internal/pkg/handler/create.go

```diff
@@ -13,17 +12,7 @@ type ResourceCreatedHandler struct {
 // Handle processes the newly created resource
 func (r ResourceCreatedHandler) Handle() error {
 	if r.Resource == nil {
-		logrus.Errorf("Error in Handler")
-	} else {
```

Comment:
why did we remove else section? why was it put in first place?

Reply:
@rasheedamir , Else section was put there to show a log statement that a new resource (i.e. secret or configmap) has been added. But it does not add any value to the reloader functionality because reloader should only show the logs when a resource has been updated.